helpdesk-message-proxy-service/src/main/java/org/symphonyoss/symphony/bots/helpdesk/messageproxy/service/TicketService.java

```diff
@@ -132,21 +139,38 @@ public void sendTicketMessageToAgentStreamId(Ticket ticket, SymMessage message)
 
     try {
       SymUser symUser = symphonyClient.getUsersClient().getUserFromId(message.getFromUserId());
-      builder.username(symUser.getDisplayName());
+      String username = symUser.getDisplayName();
+      builder.username(username);
       builder.company(symUser.getCompany());
+      builder.question(getQuestionFromMessage(message, username));
     } catch (UsersClientException e) {
       LOGGER.error("Could not get user info: ", e);
+      builder.question(getQuestionFromMessage(message, "Client"));
     }
 
-    builder.question(message.getMessageText());
-
     try {
       symphonyClient.getMessagesClient().sendMessage(stream, builder.build());
     } catch (MessagesException e) {
       LOGGER.error("Could not send ticket message to agent stream ID: ", e);
     }
   }
 
+  private String getQuestionFromMessage(SymMessage message, String username) {
+    if (SymMessageUtil.isChime(message)) {
+      return username + " sent a chime!";
```

Comment:
Please create a constant field for this text and use String.format(CHIME_MESSAGE, username) to fulfill the username.

```
private static final CHIME_MESSAGE = "%s sent a message";
```

Follow the same approach in other scenarios.